File: CitizenDemo-DotNetCore/src/CitizenDemo.ResourceAPI/Models/Resource.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace CitizenDemo.ResourceAPI.Models
{
    public class Resource
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        [BsonElement ("_id")]
        public string? InternalId { get; set; }

        [BsonElement("resourceId")]
        public string? ResourceId { get; set; }

        [BsonElement("citizenId")]
        public string? CitizenId { get; set; }

        [BsonElement("name")]
        public string? Name { get; set; }

        [BsonElement("status")]
        public string? Status { get; set; }
        public override string ToString()
        {
            return string.Format(
                "Resource: ResourceId: {0}, CitizenId: {1}, Name: {2}, Status: {3}",
                ResourceId, CitizenId, Name, Status
                );
        }
    }
}

File: CitizenDemo-Java/src/citizenapi/src/main/java/com/citizendemo/citizenapi/models/Resource.java
package com.citizendemo.citizenapi.models;

public class Resource {
    public String resourceId;
    public String citizenId;
    public String name;
    public String status;

    public Resource() {
    }

    public Resource(String resourceId, String citizenId, String name, String status) {
        this.resourceId = resourceId;
        this.citizenId = citizenId;
        this.name = name;
        this.status = status;
    }

    public String toJSONString() {
        return "{" +
                "\"resourceId\":\"" + this.resourceId + "\"," +
                "\"citizenId\":\"" + this.citizenId + "\"," +
                "\"name\":\"" + this.name + "\"," +
                "\"status\":\"" + this.status + "\"" +
                "}";
    }
}